src/types.ts

```diff
@@ -19,3 +19,5 @@ export interface HierarchyNode {
     path: string,
     children: Array<HierarchyNode>
 }
+
+export type DropHandlerFunc =(files: Array<File>, resetScene: boolean) => void;
```

Comment:
Feels like this should be declared in `drop-handler.ts`.

Reply:
done